### DISCUSSION TITLE
Sonoff USB 3.0 and Homematic CCU = Problem

### DISCUSSION THREAD

**Discussion by LordNicon79** [OpenCCU/OpenCCU#1980] [🇩🇪 Fragen&Antworten]
Hi, 
I hope I'm right with my problem here. I have HA on my RPI4. I connected a Sonoff Zigbee 3.0 USB stick and a HomematicIP RFUSB stick. First of all, the Sonoff is found perfectly and works perfectly and then I install Homematic CCU and set up the USBStick. After the next reboot, the Sonoff USB STick will no longer work/start.
If I then deactivate Homematic CCU again, everything is ok again and the Sonoff Stick runs.
The protocol said:
Error setting up entry Sonoff Zigbee 3.0 USB Dongle Plus - Sonoff Zigbee 3.0 USB Dongle Plus for zha
15:45:05 – (FEHLER) components/zha/core/gateway.py
Couldn't start ZNP = Texas Instruments Z-Stack ZNP protocol: CC253x, CC26x2, CC13x2 coordinator (attempt 1 of 3)
15:45:05 – (WARNUNG) Zigbee Home Automation - Die Nachricht ist zum ersten Mal um 15:44:39 aufgetreten und erscheint 3 mal

Does anyone happen to have an idea or knows the problem? Unfortunately I am still too inexperienced to crack the problem

Thanks for your help!  :-)

**Comment by Powerbarzi** [comment]
Hi, got the same problem yesterday, only that I use the ConbeeII Zigbee instead of the Sonoff. Have you already found something to fix the error?

**Comment by LordNicon79** [comment]
Hi,
sorry, I found no fix until now. I use a separat ccu3. I will wait for a new Version of HA an try it again.

16.11.2022 10:22:10 Powerbarzi ***@***.***>:

**Comment by IAmWebSA** [comment]
Same issue here.

Any solution available?

Setting tty config for USB somewhere?
